apps/test/unit/code-studio/components/progress/LessonLockDialogTest.jsx

```diff
@@ -1,112 +1,188 @@
 import React from 'react';
-import {shallow} from 'enzyme';
+import {mount} from 'enzyme';
 import {expect} from '../../../../util/reconfiguredChai';
 import sinon from 'sinon';
 import {UnconnectedLessonLockDialog as LessonLockDialog} from '@cdo/apps/code-studio/components/progress/LessonLockDialog';
 import {LockStatus} from '@cdo/apps/code-studio/lessonLockRedux';
 
+// This * import allows us to stub out the SectionSelector component
+import * as SectionSelector from '@cdo/apps/code-studio/components/progress/SectionSelector';
+
+const fakeSectionId = 42;
+
 const MINIMUM_PROPS = {
-  isOpen: false,
```

Comment:
In changing the default case from closed to open here, are we no longer testing the opening of the dialog?

Reply:
Good question!  I changed this because I thought it was more useful to test the dialog open for the majority of the unit tests but then added a new case to test the dialog closed.